pkg/broker/logic.go

```diff
@@ -141,7 +143,29 @@ var _ BusinessLogic = &Implementation{}
 
 func (b *Implementation) GetCatalog(w http.ResponseWriter, r *http.Request) (*osb.CatalogResponse, error) {
 	// Your catalog business logic goes here
-	return nil, nil
+	response := &osb.CatalogResponse{}
```

Comment:
Note: this is just a stopgap.  I'd like to move things around a little bit in a follow-up to get all the user code into one place

Reply:
Agreed! Maybe a separate pkg even?

Reply:
Exactly - I'm thinking pkg/user should have stub implementations of everything - CLI args, flags, business logic for broker.  That way if you have to color outside the lines frequently we get a cue that the base layer may need to change.